### PR TITLE
Add RTM icon

### LINKED ISSUE #8095: [COLLECT] Public transit networks
- [x] Île-de-France Mobilités (#8062)
- [x] RTM (#8066)
- [x] CTS (#8072)
- [x] BVG (#8073)
- [ ] FGC
- [ ] PANYNJ

Not a duplicate of #2822 since these are public transit authorities/associations managing the cities or regions public transit and not the subway or bus network itself.

### PR DESCRIPTION
**Pull request by jamu3l**
rtm

Closes #8095

**Similarweb rank:** 61,282
  

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
I picked the hex value from the logo in the Wikipedia article. They use different colours on their website and their social media logos and there is no brand guidelines available.

### CODE REVIEW

**Review by dirien — COMMENTED** [review]
Hi @jamu3l,

thanks for the contribution. Can you please open an issue for this PR too? 

Thanks!

**Comment by jamu3l** [comment]
Hey @dirien, I opened an issue (#8095) for this PR and #8062, #8072 and #8073.

**Review by dirien — CHANGES_REQUESTED** [review]
hi @jamu3l,

can you check the precision of the icon? Maybe up to 4?

I see some differences!
image

**Comment by runxel** [comment]
Hi @jamu3l, are you still on this? :)

File: _data/simple-icons.json [hunk 1]
```diff
@@ -9790,6 +9790,19 @@
             "hex": "00BCF6",
             "source": "https://www.rtl2.de/"
         },
+        {
+            "title": "RTM",
+            "hex": "244FA2",
```
Comment (PratyushJayachandran) [inline _data/simple-icons.json:9795]:
```suggestion
            "hex": "36474F",
```
Picked color from the SVG logo on their website https://www.rtm.fr

File: icons/rtm.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>RTM</title><path d="M5.584 9.053H1.98l-.487 1.414h3.408c.54 0 .594.25.51.494-.084.244-.18.322-.719.322h-3.43L0 14.947h1.917l.892-2.455c.263 0 .33.024.436.257l1.043 2.198h2.084l-1.148-2.46c1.368-.15 1.985-.823 2.206-1.699.21-.838-.173-1.735-1.846-1.735m8.876 0-.486 1.414h-1.94l-1.54 4.478H8.458l1.54-4.478H7.915l.487-1.413h6.06zm1.83 0c1.247 0 1.564.824 1.193 1.904l-1.369 3.988h-2.036l1.368-3.984c.094-.273-.05-.494-.322-.494H14.5l.485-1.413h1.305zm3.186 0c1.247 0 1.564.824 1.193 1.904l-1.37 3.988-2.035.001 1.368-3.985c.093-.273-.05-.494-.323-.494h-.479l.486-1.413h1.16zm3.185 0c1.248 0 1.564.824 1.193 1.904l-1.369 3.988H20.45l1.37-3.984c.092-.273-.052-.494-.324-.494h-.479l.485-1.413h1.16z"/></svg>
```
Comment (PratyushJayachandran) [inline icons/rtm.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>RTM</title><path d="M5.584 9.054H1.98l-.487 1.413h3.408c.54 0 .594.25.51.494-.084.244-.18.323-.719.323h-3.43L0 14.946h1.917l.892-2.454c.263 0 .33.024.436.257l1.043 2.197h2.084l-1.148-2.46c1.368-.15 1.985-.822 2.206-1.698.211-.837-.173-1.734-1.846-1.734m2.817 0-.487 1.413H10l-1.541 4.48 2.036-.001 1.54-4.479h1.94l.486-1.413H8.4zm7.89 0h-1.306l-.485 1.414h.624c.272 0 .416.22.322.493l-1.368 3.985h2.036l1.37-3.988c.37-1.08.053-1.905-1.194-1.904m3.186 0h-1.16l-.486 1.414h.48c.271 0 .415.22.322.493l-1.368 3.985H19.3l1.37-3.988c.37-1.08.053-1.904-1.194-1.904m3.185 0h-1.16l-.485 1.414h.479c.272 0 .416.22.323.493l-1.369 3.985h2.036l1.37-3.988c.37-1.08.054-1.904-1.194-1.904"/></svg>
```

SVG available on their website header https://www.rtm.fr

File: _data/simple-icons.json [hunk 3]
```diff
@@ -9790,6 +9790,19 @@
...
+            "source": "https://commons.wikimedia.org/wiki/File:Logortmcdm.svg",
```
Comment (PratyushJayachandran) [inline _data/simple-icons.json:9796]:
```suggestion
            "source": "https://www.rtm.fr",
```

**Review by PratyushJayachandran — COMMENTED** [review]
I am taking up this PR as it seems abandoned

**Comment by PratyushJayachandran** [comment]
> I am taking up this PR as it seems abandoned

@simple-icons/icon-review can someone review this?
rtm

**Review by adamrusted — APPROVED** [review]

**Pull request merged**